doc/pages/command-parsing.asciidoc

```diff
@@ -1,26 +1,24 @@
 = Command Parsing
 
-Kakoune commands, either loaded from a script, or written in the command
-prompt are parsed according to the following rules:
+Kakoune commands, either loaded from a script or written in the command prompt, are parsed according to the following rules:
 
 == Basic parsing
 
-- Commands are separated by `;` or end of lines
+- Commands are delimited by a `;` or an end of line
 
-- Words (command names and parameters) are separated by whitespaces
+- Words (command names and parameters) are delimited by whitespaces
 
 == Quoted Strings
 
-If a word starts with `'`, `"` or `%X` with `X` a non nestable
-punctuation character it is parsed as a quoted string whose delimiter is,
-respectively, `'`, `"` or `X`.
+If a word starts with `'`, `"`, or `%X` with `X` a non-nestable punctuation character (see below for nestable characters),
+it is parsed as a quoted string whose delimiter is, respectively, `'`, `"`, or `X`.
 
-A quoted string contains every character (including whitespaces) until
-its closing delimiter. If its closing delimiter is doubled, then it is
-considered to be part of the string content as a single delimiter.
+A quoted string contains every character (including whitespaces).  Doubling a closing delimiter escapes it.
+Thus, for example, entering two closing delimiting characters at the end of a quoted string will render one of the characters literally;
```

Comment:
This commit corrects a mistake introduced in a previous commit.
Reviewers don't need to know about this mistake, so to make your changes easier to read, this commit should be squashed into that previous commit.
Here's a tutorial on how to do that: https://git-rebase.io/

Reply:
I tried to do something like this, but ran into problems. I'm new to coding. Thanks for the reference.